src/mono/mono/utils/options-def.h

```diff
@@ -58,6 +58,7 @@ DEFINE_BOOL_READONLY(readonly_flag, "readonly-flag", FALSE, "Example")
 */
 
 DEFINE_BOOL(wasm_exceptions, "wasm-exceptions", FALSE, "Enable codegen for wasm exceptions")
+DEFINE_BOOL(aot_lazy_assembly_load, "aot-lazy-assembly-load", FALSE, "Load assemblies referenced by AOT images lazily")
```

Comment:
Should we make the default true for HOST_ANDROID, HOST_WASM, HOST_DARWIN && !HOST_OSX?

Reply:
I think platforms should set it themselves.

Reply:
Or maybe we can set it in the runtime startup code.